test/addons/rook-cephfs/test

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from drenv import kubectl
+
+PVC_NAME = "cephfs-pvc"
+SNAP_NAME = "cephfs-snap"
+
+
+def test_provisioning(cluster):
+    print(f"Deploying pvc {PVC_NAME} in cluster '{cluster}'")
+    kubectl.apply(
+        f"--filename={PVC_NAME}.yaml",
+        "--namespace=rook-ceph",
```

Comment:
Maybe use a namespace for the test, so if the test fails, we don't leave leftovers in the rook namespace? 

I think we have the same issue in rbd-mirror, but lets not add more of these issues.

Reply:
Done